src/helpers/documentContainerHelper.js

```diff
@@ -25,6 +25,9 @@ export const updateContainerWidth = (prevProps, props, container) =>  {
   if (leftPanelOpened) {
     shrinkContainerWidthBy(leftPanelWidth, container);
     container.style.marginLeft = `${leftPanelWidth}px`;
+  } else if (props.isLeftPanelOpen) {
+    container.style.marginLeft = `${leftPanelWidth}px`;
+    container.style.width = `calc(100% - ${leftPanelWidth}px)`;
```

Comment:
Is moving the resize bar working fine on your IE or Edge?
I tried on my computer and when I move the resize bar, I saw the position for the scroll bar is shifted as well. I suspect it's because the width of the documentContainer isn't correct?

Reply:
It looks like the css transition is causing the scroll bar to jump around. I disable it for IE and Edge and it seem to fix the issue (make resize not as smooth in IE and Edge but the scroll bar jumping is gone ).